tox.ini

```diff
@@ -0,0 +1,13 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[tox]
+envlist = py{35,36,37}
+
+[testenv]
+description = Unit tests
```

Comment:
Having this definition here is somewhat dangerous, it could lead people to think that they should primarily use tox to run the tests, and then they execute only the table-generator tests.

Is tox actually necessary for testing on Travis? Wouldn't it be possible to directly execute `nosetests` from `.appveyor.yml`? If not, can we make tox execute all our tests by default, and pass an argument from `.appveyor.yml` to run only the table-generator tests?

Reply:
Tox doesn't seem to be necessary, however [AppVeyor uses tox in their documentation on Python testing](https://www.appveyor.com/docs/lang/python/), so I took the same approach. 
I've tried executing the tests directly from `.appveyor.yml`, which (after some adjustments) [seems to be working now](https://ci.appveyor.com/project/PhilippWendler/benchexec). For Python 3.8, this requires the `--traverse-namespace` option, otherwise the tests won't be found (like for example [here](https://ci.appveyor.com/project/PhilippWendler/benchexec/builds/35291867/job/arux4n6s62rmjo05) - the same problem occurs locally on my Windows machine without this option too). I've added this option in the `setup.cfg` file as it should make no difference for lower Python versions and like this there won't be any more problems regarding this topic in the future. If there is something against it, of course I can also remove it from there and add it in the `.appveyor.yml` file instead.

I suppose we will then keep the setup without tox, right? If so, I will remove the tox file from the project.

Reply:
The option seems fine. In general it is a good idea to add the reason for having a non-obvious option (basically what you explained above, maybe shortened) either in the respective commit message or as a comment next to it, otherwise it will be hard to find later on.

Yes, I see no reason for using tox.